.github/workflows/release.yml

```diff
@@ -1,113 +1,307 @@
+          git config --global user.email "$(git --no-pager log --format=format:'%ae' -n 1)"
+
+      - name: Deploy npm
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |
+          cd build/npm
+          npm version $GAUGE_VERSION
+          npm publish --access=public
+
+  deb-rpm-release:
+    needs: [github-release]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Switch to $GITHUB_REF
+        uses: actions/checkout@v1
+
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+
+      - name: Set up Ruby 2.6.x
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
+
+      - name: Install fpm
+        run: |
+          gem install --no-document fpm
+
+      - name: Prepare deb artifact
+        run: |
+          build/mkdeb.sh
+
+      - name: Deploy debian
+        env:
+          REPO: gauge-deb
+          PACKAGE: gauge
+          DISTRIBUTIONS: stable
+          COMPONENTS: main
+          USER: ${{ secrets.BINTRAY_API_USER }}
+          API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+          PASSPHRASE: ${{ secrets.BINTRAY_GPG_SIGNING_PASSPHRASE }}
+        run: |
+          cd deploy
+          sh ../build/publish_deb_to_bintray.sh
+
+      - name: Deploy rpm
+        env:
+          REPO: gauge-rpm
+          PACKAGE: gauge
+          BINTRAY_PACKAGE: gauge-stable
+          USER: ${{ secrets.BINTRAY_API_USER }}
+          API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+          PASSPHRASE: ${{ secrets.BINTRAY_GPG_SIGNING_PASSPHRASE }}
+        run: |
+          cd deploy
+          for artifact in `ls gauge-*.deb`; do
+            fpm -t rpm -s deb $artifact
+          done
+          for artifact in `ls gauge-*.rpm`; do
+            output_package=`echo "$artifact" | sed 's/-[0-9].x/-all.x/' | sed 's/-[0-9].i/-all.i/'`
+            mv -T $artifact $output_package
+          done
+          ls
+          sh ../build/publish_rpm_to_bintray.sh
+
+  choco-release:
+    needs: [github-release]
+    runs-on: windows-latest
+    steps:
+      - name: Switch to $GITHUB_REF
+        uses: actions/checkout@v1
+
+      - name: Set gauge version
+        run: |
+          $gauge_version=($Env:GITHUB_REF -replace "[a-zA-Z//]+", '' )
+          echo "::set-env name=GAUGE_VERSION::$gauge_version"
+
+      - name: Clone chocolatey-packages
+        run: |
+          git clone https://github.com/getgauge/chocolatey-packages
+
+      - name: Deploy to Gauge $GAUGE_VERSION to choco
+        shell: pwsh
+        env:
+          API_KEY: ${{ secrets.NUGET_API_KEY }}
+        run: |
+          cd chocolatey-packages
+          $Env:GAUGE_VERSION | Set-Content version
+          cd gauge
+          ./package.ps1
+          choco push "gauge.$Env:GAUGE_VERSION.nupkg" -k $Env:API_KEY
+
+  brew-release:
+    needs: [github-release]
+    runs-on: macos-latest
```

Comment:
this one does not need `macOS` either, we don't seem to invoke `brew` command, and other scripts seem like they can run on linux